src/Tizen.NUI.Components/Controls/Navigation/DialogPage.cs

```diff
@@ -294,7 +294,7 @@ public static void ShowAlertDialog(string title, string message, params View[] a
         /// </summary>
         /// <param name="anchor">The anchor view where menu is displayed.</param>
         /// <param name="items">The menu items.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
why are you getting menu items instead of menu?

Reply:
It is to support both "Anchor" and "Anchor Position".

Since Menu supports "Anchor Position", to support "Anchor" user needs to set something more with their menu.
So I tried to provide the easier way to show menu with both "Anchor" and "Anchor Position".

Reply:
AnchorPosition is replaced with Anchor.
HorizontalPositionToAnchor and VerticalPositionToAnchor are added for Anchor.